NDTensors/ext/NDTensorsMetalExt/typeparameteraccessors.jl

```diff
@@ -0,0 +1,12 @@
+# # `TypeParameterAccessors.jl` overloads.
+using NDTensors.TypeParameterAccessors: Position, default_parameter, set_parameter
+default_parameter(::Type{<:MtlArray}, ::Position{1}) = Float32
+default_parameter(::Type{<:MtlArray}, ::Position{2}) = 1
+default_parameter(::Type{<:MtlArray}, ::Position{3}) = Metal.DefaultStorageMode
```

Comment:
I guess these will now be `default_parameter(::Type{<:MtlArray}, ::typeof(eltype)) = Float32`, etc.?

Reply:
I believe this is updated across the board. Thanks